torch/csrc/jit/init.cpp

```diff
@@ -219,6 +223,24 @@ void initJITBindings(PyObject *module) {
         return py::bytes(reinterpret_cast<const char*>(data.get()), size);
       });
 
+  m.def("_jit_get_operation", [](const std::string& qualified_name) {
+    try {
+      auto symbol = Symbol::fromQualString(qualified_name);
+      auto operations = getAllOperatorsFor(std::move(symbol));
+      AT_CHECK(!operations.empty(), "No such operator ", qualified_name);
+      AT_CHECK(
+          operations.size() == 1,
+          "Found ", operations.size(), " overloads for operator ",
+          qualified_name, "! Overloads are not supported from Python.");
+      auto& op = operations[0];
+      return py::cpp_function([op](py::args args, py::kwargs kwargs){
```

Comment:
nit: it is bit hard to see here that what we are caputuring is a `shared_ptr<Operator>`, might be clearer with
```
std::shared_ptr<Operator> op = operations[0];
```